frontend/src/app/app.module.ts

```diff
@@ -8,17 +8,33 @@ import { AngularFireModule } from '@angular/fire';
 import { AngularFirestoreModule } from '@angular/fire/firestore';
 import { environment } from '../environments/environment';
 import { ImagesComponent } from './images/images.component';
+import { ClarityModule } from '@clr/angular';
```

Comment:
super small comment (literally means nothing) but some projects like to have these imports ordered in some sort of fashion. The order I kind of prefer is 3p libraries, project core dependencies, and then internal app dependencies. I'll submit a PR with some of my comments to show what I'm talking about. 

Again...literally _does not matter_ and is purely a preferential thing.

Reply:
I'm intrigued, I never happened to follow an import structure before, so this is a good rule of thumb I'm going to follow.